packages/react-native-reanimated/apple/worklets/WorkletsModule.mm

```diff
@@ -0,0 +1,48 @@
+#import <React/RCTBridge+Private.h>
+#import <worklets/WorkletRuntime/RNRuntimeWorkletDecorator.h>
+#import <worklets/apple/WorkletsModule.h>
+
+@interface RCTBridge (JSIRuntime)
+- (void *)runtime;
+@end
+
+@interface RCTBridge (RCTTurboModule)
+- (std::shared_ptr<facebook::react::CallInvoker>)jsCallInvoker;
+- (void)_tryAndHandleError:(dispatch_block_t)block;
+@end
+
+@implementation WorkletsModule {
+  std::shared_ptr<NativeWorkletsModule> nativeWorkletsModule_;
+  bool _isBridgeless;
+}
+
+- (std::shared_ptr<NativeWorkletsModule>)getNativeWorkletsModule
+{
+  return nativeWorkletsModule_;
+}
+
+@synthesize moduleRegistry = _moduleRegistry;
+
+RCT_EXPORT_MODULE(WorkletsModule);
+
+- (void)invalidate
```

Comment:
Why do you call those supers?

Reply:
Fixed in [22793d2](https://github.com/software-mansion/react-native-reanimated/pull/6539/commits/22793d24d0b5455ef915942911a82043dc419925)